Assets/SC_bALL.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_bALL : MonoBehaviour
```

Comment:
The name of the class is unintuitive and incomprehensible. It would be better to name the class using PascalCase.
```suggestion
public class BallScript : MonoBehaviour
```